sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -111,6 +111,10 @@ page at http://checkstyle.sourceforge.net/config.html -->
       <property name="illegalPkgs" value="avro.shaded, com.google.api.client.repackaged, com.google.appengine.repackaged, io.netty.util.internal"/>
     </module>
 
+    <module name="RedundantModifier">
+      <property name="tokens" value="VARIABLE_DEF, ANNOTATION_FIELD_DEF, CLASS_DEF, ENUM_DEF"/>
```

Comment:
I'm finding that validating `METHOD_DEF` is also goodness. Will add in another commit shortly..